osd/limited_support/GovcloudLimitedSupportOneClusterPerVPC.json

```diff
@@ -0,0 +1,8 @@
+{
+  "severity": "Warning",
+  "kind": "LimitedSupportReason",
+  "summary": "Cluster is in Limited Support due to unsupported network configuration",
+  "details": "SRE has observed that there is >1 cluster inside a single VPC in your AWS account. VPC sharing is currently unsupported by ROSA Classic on Govcloud, please reach out to customer support for more details.",
```

Comment:
"details": "SRE has observed that there is >1 cluster within a single VPC in your AWS account. VPC sharing is not supported within AWS GovCloud Regions for ROSA Classic. Please use the ROSA UI or CLI to remove one of these cluster from VPC ${VPCID}.  If this cluster is the one remaining in the VPC when completed, please open a ServiceNow case for SRE to review.",